pylib/mozhginfo/tests/test_pushlog_client.py

```diff
@@ -75,52 +78,56 @@ def setUp(self):
         self.end_revision = '2a193b7fd62171da1357e6c1f93453359e54206b'
 
     @patch('requests.get', return_value=mock_response(LIST_REVISION, 200))
-    def test_query_revisions_range(self, get):
-        self.assertEqual(
-            query_revisions_range(self.repo_url, self.start_revision,
-                                  self.end_revision, version=2, tipsonly=1),
-            ['8850aa0f93453d6a4472f8905271aba33f8b68d5', 'eb15e3f893453d6a4472f8905271aba33f8b68d5',
-             '1c5b4332e2f1b73fe03977b69371e9a08503bff3', '724f0a71d62171da1357e6c1f93453359e54206b']
-        )
+    @patch('mozhginfo.pushlog_client.query_push_by_revision',
+           return_value=push)
+    def test_query_pushes_by_revision_range(self, get, query_push_by_revision):
+        pushes = query_pushes_by_revision_range(self.repo_url, self.start_revision,
+                                                self.end_revision, version=2, tipsonly=1)
```

Comment:
Could you please call the function explicitely listing the various arguments it takes?
> repo_url=self.repo_url, 
> from_revision=self.start_revision

This is a trick to prevent in the future changing the signature of a function and causing unexpected intents.
For example, the arguments in the function changes order, however, we forget fixing the tests.
Such an issue would only be discovered upon a bug being faced by a system or a user using it.